src/equations/bbm_bbm_1d.jl

```diff
@@ -148,15 +151,15 @@ end
 @inline function prim2cons(q, equations::BBMBBMEquations1D)
     eta, v = q
 
-    h = eta + equations.D
+    h = eta - equations.eta0 + equations.D
```

Comment:
Although `equations.eta0` needs to be 0, I use it in the conversions such that it can be generalized more easily in the future if required and such that it matches the general definitions. IMO, this is a bit more understandable, but it might be less understandable because it is always 0.

Reply:
👍